python/scenario/report/_save.py

```
@@ -246,57 +250,76 @@ def save_redteam_report(
+                success=success,
                 reasoning=_reasoning,
                 failed_criteria=_failed,
                 transcript=_transcript_for_prompt(messages),
                 model=analyzer_model or red_team.metaprompt_model,
             )
             if analysis.get("failing_turn_index") is None and failing_turn_hint is not None:
                 analysis["failing_turn_index"] = failing_turn_hint
+            # _analyze returns "" when the model's break_severity was
+            # unrecognized — that is an analysis failure, not a "none".
+            analysis_failed = analysis.get("break_severity") == ""
         except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)